packages/preact-iso/router.js

```diff
@@ -19,8 +19,8 @@ const UPDATE = (state, url, push) => {
 };
 
 export const exec = (url, route, matches) => {
-	url = url.trim('/').split('/');
-	route = (route || '').trim('/').split('/');
+	url = url.split('/').filter(Boolean);
```

Comment:
Just checked because I was curious: The issue is that `trim()` has [no arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) and only trims whitespace. So the original code was wrong to begin with.

Reply:
lol yeah that's my bad.